Gemfile

```diff
@@ -3,6 +3,7 @@ ruby '2.2.2'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.3'
+
```

Comment:
What were the changes you made? I do not see anything but a new line in the Gemfile.